### DISCUSSION TITLE
xhtml to docx: footnotes

### DISCUSSION THREAD

**Discussion by zsofiamacho** [jgm/pandoc#10143] [Q&A] [answered]
Hi,

I am mostly looking for confirmation that this feature indeed doesn't work ... 
If I use markdown to docx, the footnotes use the "native" footnote functionality in word. So, if I were to edit the docx document, and add another footnote between 1 and 2, the numbering would consequently change.

However, if I try markdown > xhtml > docx (to ensure correct and expected xhtml markup), the footnotes are "merely" superscripted links, pointing to a list that is separated from the main text by a line.

Is there anything I can do to make pandoc treat sup+a+specific class in a html as it amazingly does for markdown?

**Comment by jgm** [comment, marked as answer]
Relevant issue: #5294
